docs: Remove aarch64 unsupported note for Homebrew on Linux #1 (#19680)
Homebrew/brew: docs/Homebrew-on-Linux.md

- **Linux** 3.2 or newer
- **Glibc** 2.13 or newer

> - **64-bit x86_64** or **64-bit aarch64** (including Apple Silicon) CPU's

Reviewer: ```suggestion
```

  sudo pacman -S base-devel procps-ng curl file git
  ```

> ### ARM32 (unsupported)

Reviewer: ```suggestion
### ARM32 (Tier 3 Support)
```

> Homebrew can run on 32-bit ARM (e.g. Raspberry Pi and others), but as they lack bottles (binary packages) they are unsupported. Pull requests are welcome to improve the experience on ARM platforms.

Reviewer: ```suggestion
Homebrew can run on 32-bit ARM (e.g. Raspberry Pi and others), but as they lack bottles (binary packages) they are a [Tier 3 supported platform](https://docs.brew.sh/Support-Tiers#tier-3)
```
